### ISSUE TITLE
mn client doesn't work with OpenJDK Runtime Environment (build 11.0.4+11-post-Ubuntu-1ubuntu218.04.3) after sdk upgrade

### ISSUE THREAD

**Issue by dcalap** [micronaut-projects/micronaut-core#2533]
Micronaut client doesn't work anymore after upgrading with sdkman.

### Steps to Reproduce

1. Run `sdk upgrade micronaut`:
![image](https://user-images.githubusercontent.com/6464175/71018983-cdce0d00-20f9-11ea-871c-635f2b3572c3.png)

2. Run micronaut client with  `mn`:
![image](https://user-images.githubusercontent.com/6464175/71019041-f0602600-20f9-11ea-8f46-643f776fbde3.png)

### Expected Behaviour
Micronaut client starts properly

### Actual Behaviour
An error message appears:
```
Unrecognized option: --add-opens=java.base/java.io=ALL-UNNAMED
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

### Environment Information

- **Operating System**: Ubuntu 18.04.3 LTS
- **Micronaut Version:** 1.2.8
- **JDK Version:** 
openjdk 11.0.4 2019-07-16
OpenJDK Runtime Environment (build 11.0.4+11-post-Ubuntu-1ubuntu218.04.3)
OpenJDK 64-Bit Server VM (build 11.0.4+11-post-Ubuntu-1ubuntu218.04.3, mixed mode, sharing)

**Comment by jameskleeh** [comment]
What shell are you using?

**Comment by jameskleeh** [comment]
I attempted to reproduce with
```openjdk 11.0.5 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-post-Ubuntu-0ubuntu1.118.04)
OpenJDK 64-Bit Server VM (build 11.0.5+10-post-Ubuntu-0ubuntu1.118.04, mixed mode, sharing)``` and could not reproduce. I tried with both sh and bash

**Comment by dcalap** [comment]
Right. I'm using zsh.

**Comment by reyjexter** [comment]
I just starting playing around with Micronaut and got this error as well on Ubuntu 18.04. I also just downloaded the binary from this link and just manually set the path: https://micronaut.io/download.html. 

I'm also just using the default Terminal from from Ubuntu 18.04

I wonder if there's any workaround to this issue. Thanks


`
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic
`

`
mn --version
Unrecognized option: --add-opens=java.base/java.io=ALL-UNNAMED
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
`

`
java --version
openjdk 11.0.7 2020-04-14
OpenJDK Runtime Environment (build 11.0.7+10-post-Ubuntu-2ubuntu218.04)
OpenJDK 64-Bit Server VM (build 11.0.7+10-post-Ubuntu-2ubuntu218.04, mixed mode, sharing)
`

**Comment by ilopmar** [comment]
Although I know it's not exactly the same, I've tried with Docker because in my Linux Mint I'm not able to reproduce your issue.

```
docker run -it --rm ubuntu:18.04 bash

# Install dependencies
apt update
apt install zip unzip curl

# Install SDKMan
curl -s "https://get.sdkman.io" | bash

# Install OpenJDK 11
sdk install java 11.0.7.hs-adpt

# Install Micronaut 1.3.6
sdk install micronaut 1.3.6

java -version
openjdk version "11.0.7" 2020-04-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.7+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.7+10, mixed mode)

 mn --version
| Micronaut Version: 1.3.6
| JVM Version: 11.0.7
```

**Comment by reyjexter** [comment]
@ilopmar 
After changing version, I somehow got past the error message I previously posted but I also ended up with another error which maven repository are not being found. 

The ultimate solution was to upgrade to 2.0.0.M3 and install it using sdkman instead of adding the binaries to path which I also tested and the openjdk 11 version I posted was still workgin.

So I also tried 1.3.6 with sdkman installation and I was getting an error which says that some files under maven folder was missing.
